docs/developer-guides/contributing.md

```diff
@@ -0,0 +1,43 @@
+---
+id: contributing
+---
+
+# Contributing
+
+## Contributor License Agreement
+
+This project welcomes contributions and suggestions.  Most contributions require you to agree to a
+Contributor License Agreement (CLA) declaring that you have the right to, and actually do, grant us
+the rights to use your contribution. For details, visit https://cla.opensource.microsoft.com.
+
+When you submit a pull request, a CLA bot will automatically determine whether you need to provide
+a CLA and decorate the PR appropriately (e.g., status check, comment). Simply follow the instructions
+provided by the bot. You will only need to do this once across all repos using our CLA.
+
+This project has adopted the [Microsoft Open Source Code of Conduct](https://opensource.microsoft.com/codeofconduct/).
+For more information see the [Code of Conduct FAQ](https://opensource.microsoft.com/codeofconduct/faq/) or
+contact [opencode@microsoft.com](mailto:opencode@microsoft.com) with any additional questions or comments.
+
+## How to Contribute
+
+### Contribute New Feature
+
+MS-AMP is an open-source project. Your participation and contribution are highly appreciated. There are several important things you need know before contributing new feature to this project:
+
+#### What content can be added to MS-AMP
+
+1. Bug fixes for existing features.
+1. Performance improvement.
+1. New features such as support for new distributed training framework.
+
+   If you would like to contribute a new feature on MS-AMP, please submit your proposal first. In [GitHub Issues](https://github.com/azure/MS-AMP/issues) module, choose `Enhancement Request` to finish the submission. If the proposal is accepted, you can submit pull requests to origin `main` branch.
+
+#### Contribution steps
+
+If you would like to contribute to the project, please follow below steps of joint development on GitHub.
+
+1. `Fork` the repo first to your personal GitHub account.
+1. Checkout from main branch for feature development.
```

Comment:
The order number should be fixed.

Reply:
Using order number in markdown will break some rule in mklint. Always using `1` for all items has same effect in markdown.
https://azure.github.io/MS-AMP/docs/developer-guides/contributing 
![image](https://github.com/Azure/MS-AMP/assets/4133758/e65f7cb2-33fe-4b70-a601-b0bc007845a3)